### PR TITLE
Update shapes.py

### PR DESCRIPTION
**Pull request by roy537**
got you there 🐢

### CODE REVIEW

File: shapes.py [hunk 1]
```diff
@@ -81,3 +81,27 @@ def rectangle(width:int, height:int, fill: bool = False, savefile: None | str =
 
 
   print("\n".join(layers))
+
+
+def hexagon(edge_length: int = 2) -> None:
```
Comment (max-de-rooij) [inline shapes.py:86]: Function does not follow the signature of the other shapes.

File: shapes.py [hunk 2]
```diff
@@ -81,3 +81,27 @@ def rectangle(width:int, height:int, fill: bool = False, savefile: None | str =
...
+  """
+  draws a hexagon of with edges of size 'edge_length' using the turtle package
+
+  Arguments:
+    - edge_length : integer, should be equal or larger than 0 and and uneven number
```
Comment (max-de-rooij) [inline shapes.py:91]: Why should the number be uneven? Also, there is no check that `edge_length` satisfies this requirement in the function.

File: shapes.py [hunk 3]
```diff
@@ -81,3 +81,27 @@ def rectangle(width:int, height:int, fill: bool = False, savefile: None | str =
 
 
   print("\n".join(layers))
+
+
+def hexagon(edge_length: int = 2) -> None:
+  """
+  draws a hexagon of with edges of size 'edge_length' using the turtle package
```
Comment (max-de-rooij) [inline shapes.py:88]: The English here is not correct. I think you meant:
`"draws a hexagon with edges of size 'edge_lenght' using the turtle package"`

File: shapes.py [hunk 4]
```diff
@@ -81,3 +81,27 @@ def rectangle(width:int, height:int, fill: bool = False, savefile: None | str =
...
+
+  Arguments:
+    - edge_length : integer, should be equal or larger than 0 and and uneven number
+  
+  Returns:
+    - 
+  """
+  import turtle
```
Comment (max-de-rooij) [inline shapes.py:96]: Importing a package inside the function scope risks bugs. Consider importing packages at the top of the file. Also, I think you should consider including turtles as an external requirement by creating a `requirements.txt` file, and adding this package including the specific versions to it.

File: shapes.py [hunk 5]
```diff
@@ -81,3 +81,27 @@ def rectangle(width:int, height:int, fill: bool = False, savefile: None | str =
...
+  if (edge_length <= 0):
+    raise ValueError(f"edge length must be larger than 0, {edge_length} <= 0")
+
+  turtle.pendown()
+  for _ in range(6):
+      turtle.forward(edge_length)
+      turtle.left(60)
+  turtle.penup()
+
+  print(";) got you there")
```
Comment (max-de-rooij) [inline shapes.py:107]: This line seems to be unrelated to the function. Please consider removing it.

**Review by max-de-rooij — COMMENTED** [review]
Your addition does not seem to follow the general style and functionality of the package. The other functions in the package explicitly print the requested shapes, or save them to a text file, while your function uses the turtles package to draw a shape.

File: shapes.py (hunk 3 above)
Comment (roy537) [reply shapes.py:88]: I think you meant 'edge_length'

File: shapes.py (hunk 4 above)
Comment (roy537) [reply shapes.py:96]: 🐢

File: shapes.py (hunk 5 above)
Comment (roy537) [reply shapes.py:107]: turtle.left.the.chat()

**Comment by max-de-rooij** [comment]
Closing because no requested features.
